src/microbit/test/test_shim.py

```diff
@@ -0,0 +1,19 @@
+import time
+
+import pytest
+from unittest import mock
+from .. import shim
+from ..model import microbit_model
+
+
+class TestShim(object):
+    def test_sleep(self):
+        n = 100

```

Comment:
_&ast;nit&ast;_
Maybe name this variable something more descriptive?

Reply:
fixed